cmd/ctr/app/main.go

```diff
@@ -55,6 +62,38 @@ func init() {
 	}
 }
 
+func newTraceProvider(ctx context.Context, endpoint string) (*trace.TracerProvider, error) {
+	u, err := url.Parse(endpoint)
+	if err != nil {
+		return nil, err
+	}
+
+	opts := []otlptracehttp.Option{otlptracehttp.WithEndpoint(u.Host)}
+	if u.Scheme == "http" {
+		opts = append(opts, otlptracehttp.WithInsecure())
+	} else if u.Scheme == "https" {
+		// Do nothing. HTTPS is the default
+	} else {
+		return nil, fmt.Errorf("%s is not supported", u.Scheme)
+	}
+
+	exp, err := otlptracehttp.New(ctx, opts...)
+	if err != nil {
+		return nil, err
+	}
+	res, err := resource.New(ctx,
+		resource.WithHost(),
+		resource.WithAttributes(semconv.ServiceNameKey.String("ctr")),
+	)
+	if err != nil {
+		return nil, err
+	}
+	return trace.NewTracerProvider(
+		trace.WithSpanProcessor(trace.NewSimpleSpanProcessor(exp)),
```

Comment:
FYI: It is recommended to use `BatchSpanProcessor` for better performance and optimization.

Reply:
I didn't know that. Fixed in the last revision.